Makefile

```diff
@@ -115,6 +115,31 @@ functional-test-tvos:
 	rm -rf build/reports/tvosFunctionalResults.xcresult
 	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme "FunctionalTests" -destination 'platform=tvOS Simulator,name=Apple TV' -derivedDataPath build/out -resultBundlePath build/reports/tvosFunctionalResults.xcresult -enableCodeCoverage YES ADB_SKIP_LINT=YES
 
+# Runs the Edge Network (Konductor) integration tests after installing pod dependencies
+# Usage: 
+# make test-integration-upstream EDGE_ENVIRONMENT=<environment> EDGE_LOCATION_HINT=<location_hint>
+# If EDGE_ENVIRONMENT is not specified, test target will use its default value.
+.SILENT: test-integration-upstream # Silences Makefile's automatic echo of commands
+test-integration-upstream: pod-install; \
+	if [ -z "$$EDGE_ENVIRONMENT" ]; then \
+		echo ''; \
+		echo '-------------------------- WARNING -------------------------------'; \
+		echo 'EDGE_ENVIRONMENT was NOT set; the test will use its default value.'; \
+		echo '------------------------------------------------------------------'; \
+		echo ''; \
+	fi; \
+	xcodebuild test \
+	-quiet \
+	-workspace $(PROJECT_NAME).xcworkspace \
+	-scheme UpstreamIntegrationTests \
+	-destination 'platform=iOS Simulator,name=iPhone 14' \
+	-derivedDataPath build/out \
+	-resultBundlePath coverage/upstreamIntegrationTest/iosresults.xcresult \
```

Comment:
for uniformity can we move these to build/reports/iosIntegrationUpstreamResults.xcresult, this will also ensure the file is removed with `make clean`.

Reply:
getting a warning that 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, can you update the ios/tvos target to 11.0 based on latest changes from dev?